appcues/src/main/java/com/appcues/action/AppcuesActionFactories.kt

```diff
@@ -0,0 +1,44 @@
+package com.appcues.action
+
+import com.appcues.action.appcues.CloseAction
+import com.appcues.action.appcues.ContinueAction
+import com.appcues.action.appcues.LaunchExperienceAction
+import com.appcues.action.appcues.LinkAction
+import com.appcues.action.appcues.RequestReviewAction
+import com.appcues.action.appcues.SubmitFormAction
+import com.appcues.action.appcues.TrackEventAction
+import com.appcues.action.appcues.UpdateProfileAction
+import org.koin.core.parameter.parametersOf
+import org.koin.core.scope.Scope
+
+internal fun Scope.closeActionFactory(): ActionFactoryBlock {
```

Comment:
created this file for both actions and traits. trying to ensure readability on their main class (ActionRegistry and TraitRegistry)